### ISSUE TITLE
Window fails to load 3rd instance then crash

### ISSUE THREAD

**Issue by enitdev** [johnpapa/vscode-peacock#503]
**Versions (please complete the following information):**

- OS: macOS Catalina Version: 10.15.7
- VS Code version: Version: 1.68.1 (Universal)
- Peacock Version: 4.1.0

**Describe the bug**
Error: The window has crashed (reason: 'clean-exit', code: '0')

**To Reproduce**

Open new window, on third instance the window fails to load, after 10-20 seconds crash with error.

**Comment by willpower232** [comment]
does this still happen on the latest version of vscode?

**Comment by enitdev** [comment]
Yes it does. I can offer Remote Desktop if you need to see it happen IRL.

**Comment by willpower232** [comment]
I can't reproduce but a quick google loosely implies it could be a mac thing and I'm not on mac so pretty odd. Does this still happen if you disable peacock first?

**Comment by enitdev** [comment]
When Peacock is disabled I can open 3+ instances.

**Comment by johnpapa** [comment]
Thank you for reporting this! This appears to be a VS Code stability issue. Closing as stale — please open a new issue if still seeing crashes on current VS Code!
